geospaas_harvesting/ingesters.py

```diff
@@ -554,10 +554,15 @@ def _get_geometry_wkt(self, dataset):
         # point
         elif longitudes.shape == latitudes.shape:
             points = []
+            #read the fill_value just one time for usage in below loop
+            lat_fil_value = latitudes[:].fill_value if np.ma.isMaskedArray(latitudes) else None
+            lon_fil_value = longitudes[:].fill_value if np.ma.isMaskedArray(longitudes) else None
             # in this case numpy.nditer() works like zip() for
             # multi-dimensional arrays
             for lon, lat in np.nditer((longitudes, latitudes), flags=['buffered']):
                 new_point = Point(float(lon), float(lat), srid=4326)
+                if lat_fil_value in new_point.coords or lon_fil_value in new_point.coords:
+                    continue # Don't add the points that have the default value inside them
```

Comment:
You can directly test on `lon` and `lat` before instantiating the Point object.
There is no point in doing this operation if the object is not going to be used.
Also, it's better to test `lat_fil_value` against the latitude and `lon_fil_value` against the longitude rather than just checking if either of the fill values is in either of the coordinates.

Reply:
Done